assets/training/finetune_acft_hf_nlp/components/finetune/text_classification/spec.yaml

```diff
@@ -1,14 +1,14 @@
 $schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
 name: text_classification_finetune
-version: 0.0.11
+version: 0.0.12
 type: command
 
 is_deterministic: True
 
 display_name: Text Classification Finetune
 description: Component to finetune Hugging Face pretrained models for text classification task. The component supports optimizations such as LoRA, Deepspeed and ONNXRuntime for performance enhancement. See [docs](https://aka.ms/azureml/components/text_classification_finetune) to learn more.
 
-environment: azureml://registries/azureml/environments/acft-hf-nlp-gpu/versions/14
+environment: azureml:acft-hf-nlp-gpu:16
```

Comment:
why has the environment format changed from assetId to name:version?

Reply:
So it can pick up the corresponding environment from the same registry.